install_files/ansible-base/roles/app-test/tasks/main.yml

```diff
@@ -6,3 +6,5 @@
   tags: non-development,aa-complain
 
 - include: dev_setup_xvfb_for_functional_tests.yml
+
+- include: security_hardening_tests.yml
```

Comment:
This should be moved to a different location to avoid the travis playbook running these tasks which wont work in the travis environment.